exporter/datasetexporter/logs_exporter.go

```diff
@@ -65,6 +66,73 @@ func buildBody(attrs map[string]interface{}, value pcommon.Value) string {
 	return message
 }
 
+// Function maps OTel severity on the LogRecord to DataSet severity level (number)
+func otelSeverityToDataSetSeverity(log plog.LogRecord) int {
+	// If log record doesn't contain severity or we can't map it to a valid DataSet severity,
+	// we use this value (INFO) instead
+	defaultSeverityLevel := 3
```

Comment:
this constant can be extracted outside of the method.

Reply:
Agreed, will move it to the file / module level.

Reply:
Done.